static/js/generic_multiple_choice.html

```diff
@@ -0,0 +1,288 @@
+<html>
+	
+<style>
+
+.bar rect {
+  fill: steelblue;
+  shape-rendering: crispEdges;
+}
+
+.bar text {
+  fill: #fff;
+}
+
+.axis path, .axis line {
+  fill: none;
+  stroke: #000; 
+  shape-rendering: crispEdges;
+}
+
+</style>
+
+<head>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
Since we're moving to https it looks like we'll need to use **https** for all external resources as well.

Not in this pull request, though.